docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+
+{{% notice info %}} You now have created an AKS cluster with `PersistentVolumeClaim` and `PersistentVolume` to contain the WLS domain configuration files.  Using those artifacts, you have used the operator to create a WLS domain.
+{{% /notice %}}
+
+#### Deploy sample application
+
+Now that you have WLS running in AKS, you can test the cluster by deploying the simple sample application included in the repository:
+
+1. Go to the WebLogic Server Administration Console, Select "Lock & Edit".
+1. Select Deployments.
+1. Select Install.
+1. Select Upload your file(s).
+1. For the Deployment Archive, Select "Choose File".
+1. Select the file `kubernetes/samples/charts/application/testwebapp.war`.
+1. Select Next. Choose 'Install this deployment as an application'.
+1. Select Next. Select cluster-1 and All servers in the cluster.  Select Next.
+1. Accept the defaults in the next screen and select Next
+1. Select Finish.
+1. Select Activate Changes.
+
+{{%expand "Click here to view the application deployment screenshot." %}}
+![Deploy Application](../screenshot-deploy-test-app.png)
+{{% /expand %}}
+
+Next you will need to start the application:
+
+1. Go to Deployments.
+1. Select Control.
+1. Select the checkbox next to testwebapp.
+1. Select Start.
+1. Select Servicing all requests.
+1. Select Yes.
+
+After the successful deployment, go to the application through the domain1-cluster-1-lb external IP.
+
+```bash
+$ kubectl  get svc domain1-cluster-1-external-lb
+NAME                            TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)          AGE
+domain1-cluster-1-external-lb   LoadBalancer   10.0.108.249   52.224.248.40   8001:32695/TCP   30m
+```
+
+In the example, the application address is: `http://52.224.248.40:8001/testwebapp`.
+
+The test application will list the server host and server IP on the page.
+
+#### Access WebLogic Server logs
+
+The logs are stored in the Azure file share. Follow these steps to access the log:
+
+1. Go to the [Azure Portal](https://ms.portal.azure.com).
+2. Go to your resource group.
+3. Open the storage account.
+4. In the "File service" section of the left panel, select File shares.
+5. Select the file share name (e.g. weblogic in this example).
+6. Select logs.
+7. Select domain1.
+8. WebLogic Server logs are listed in the folder.
+
+{{%expand "Click here to view the WebLogic Server logs screenshot." %}}
+![WebLogic Server Logs](../screenshot-logs.png)
+{{% /expand %}}
+
+#### Clean up resources
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/clean-up-resources-body-01.txt" >}}
+
+If you created the AKS cluster step by step, run the following commands to clean up resources.
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/clean-up-resources-body-02.txt" >}}
+
+#### Troubleshooting
+
+[Troubleshooting]({{< relref "/samples/simple/azure-kubernetes-service/troubleshooting.md" >}})
```

Comment:
[Troubleshooting] -> For troubleshooting advice (or information, whichever you prefer), see [Troubleshooting]... (period)

Reply:
Will do.